### PR TITLE
Fix mDNS browser

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #3336

Also mDSN browser in PortHandler is now disabled by default. Please use options tab to enable the feature.

### CODE REVIEW

**Comment by Schlaegel** [comment]
This works for me!!!! W00H000! It's the first time 10.9 or 10.10 has worked. I tested it both with MDNS and without MDNS. Both worked. (The title is incorrectly labeled MSDN, which brings back memories of the Microsoft Developer Network and my career many decades ago.)

Thanks

**Comment by haslinghuis** [comment]
Updated labels to mDSN

**Comment by Schlaegel** [comment]
Almost. It's mDNS. Like regular DNS (Domain Name System) but with a lower case 'm' in front for multicast.

**Comment by haslinghuis** [comment]
Fixed.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
* Although i cannot test any network/wifi, i verify that this does not break any USB usage. I can connect USB with the mDNS toggle `on` or `off`. (`Ubuntu 22.04.1 LTS`)

File: src/js/port_handler.js
Comment (chmelevskij) [inline src/js/port_handler.js:91]:
```suggestion
            currentPorts = cp;
```

File: src/js/tabs/options.js
Comment (McGiverGim) [inline src/js/tabs/options.js:145]:
```suggestion
options.initUseMdnsBrowser = function() {
```

File: src/js/tabs/options.js
Comment (McGiverGim) [inline src/js/tabs/options.js:24]:
```suggestion
        TABS.options.initUseMdnsBrowser();
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/js/tabs/options.js
Comment (haslinghuis) [reply src/js/tabs/options.js:145]: Good catch !

Comment (haslinghuis) [reply src/js/tabs/options.js]: Ah it worked because line 24

File: src/js/tabs/options.js
Comment (haslinghuis) [reply src/js/tabs/options.js]: Fixed!

File: src/js/port_handler.js
Comment (haslinghuis) [reply src/js/port_handler.js]: Fixed!

**Pull request merged**
